configure

```diff
@@ -2974,9 +2982,14 @@ POSTGRES_BUILDDIR="$POSTGRES_BUILDDIR"
 
 ac_config_files="$ac_config_files Makefile.global"
 
-ac_config_headers="$ac_config_headers src/include/citus_config.h"
+ac_config_headers="$ac_config_headers src/include/citus_config.h src/include/citus_version.h"
+
 
 
+cat >>confdefs.h <<_ACEOF
+#define CITUS_BINARY_VERSION "$citus_binary_version"
```

Comment:
Doesn't this method of implementation mean I need to re-run `configure` every time the version number changes? I don't run configure very often right now and would prefer to keep it that way. _Must_ this be done at this level?